src/assets/config_files/usecases.json

```diff
@@ -33,10 +20,23 @@
             "extension": ".ipynb",
             "file": "3D6ffd5ebb-a36c-4fa4-9f44-d329c9e6d3e1",
```

Comment:
I don't have access to that file. Is it in a private Collab?
If that is the case you have 2 options: either add us as team members (@alex4200 , me, @lbologna etc) or create a public Collab so everybody can have a look.

Reply:
@antonelepfl Sorry the "3D" at the start of the UUID was part of the HTML code in the file URL. I've updated usecases.json and it should be visible now.